### ISSUE TITLE
[BUG] ERROR TypeError: Cannot set property 'submission' of null

### ISSUE THREAD

**Issue by kamaloush** [formio/angular#734]
## Environment
- Hosting type
  - [x] Local deployment
    - Version: 5.0.2
- Formio.js version: 4.12.7

## Steps to Reproduce
1. Install @formio/angular
2. Add FormioModule
3. Add the form from the readme
```html
<formio [form]='{
    "title": "My Test Form",
    "components": [
        {
            "type": "textfield",
            "input": true,
            "tableView": true,
            "inputType": "text",
            "inputMask": "",
            "label": "First Name",
            "key": "firstName",
            "placeholder": "Enter your first name",
            "prefix": "",
            "suffix": "",
            "multiple": false,
            "defaultValue": "",
            "protected": false,
            "unique": false,
            "persistent": true,
            "validate": {
                "required": true,
                "minLength": 2,
                "maxLength": 10,
                "pattern": "",
                "custom": "",
                "customPrivate": false
            },
            "conditional": {
                "show": "",
                "when": null,
                "eq": ""
            }
        },
        {
            "type": "textfield",
            "input": true,
            "tableView": true,
            "inputType": "text",
            "inputMask": "",
            "label": "Last Name",
            "key": "lastName",
            "placeholder": "Enter your last name",
            "prefix": "",
            "suffix": "",
            "multiple": false,
            "defaultValue": "",
            "protected": false,
            "unique": false,
            "persistent": true,
            "validate": {
                "required": true,
                "minLength": 2,
                "maxLength": 10,
                "pattern": "",
                "custom": "",
                "customPrivate": false
            },
            "conditional": {
                "show": "",
                "when": null,
                "eq": ""
            }
        },
        {
            "input": true,
            "label": "Submit",
            "tableView": false,
            "key": "submit",
            "size": "md",
            "leftIcon": "",
            "rightIcon": "",
            "block": false,
            "action": "submit",
            "disableOnInvalid": true,
            "theme": "primary",
            "type": "button"
        }
    ]
}'></formio>
```
## Expected behavior
Form displayed
## Observed behavior
Spinner and error in devtools
![image](https://user-images.githubusercontent.com/26444823/110111400-8e4c1900-7db0-11eb-820f-efd555e6577b.png)
![image](https://user-images.githubusercontent.com/26444823/110111436-9c9a3500-7db0-11eb-90b1-2c10b5505b6a.png)

## Example
https://github.com/kamaloush/angularformio
Just clone repo, npm install and ng serve
(sorry I tried to create a stackblitz and a codesandbox project but it doesn't work)

**Comment by wag110894** [comment]
Hello @kamaloush,

Thank you for reaching out! We will get one of our developers to take a look at this issue. For reference, the ticket number is FIO-2481.

**Comment by rumeshwick** [comment]
Any luck on this ? i'm also getting the same error

**Comment by cuneytdalan** [comment]
I had the same issue when installed the package with npm but after added the @formio/angular in my project as a library under project without building its working properly. Also i can't give specific path inside tsconfig for @formio/angular. There may also be a problem with paths.

**Comment by cbc872** [comment]
Any idea when this will be fixed? It is a blocker for me

**Comment by cuneytdalan** [comment]
@cbc872  It worked on my project with these:

"@angular/cli": "^11.0.2"
"@formio/angular": "^5.0.2"
"formiojs": "^4.12.7"

**Comment by fillumina** [comment]
Unfortunately it doesn't work for me too with angular 10.0.0 and because I'm using JHipster I'm stuck on that.

**Comment by fillumina** [comment]
I have the same problem with angular 10.0.0, but it is working using
"@formio/angular": "5.0.3-rc.1"

**Comment by kamaloush** [comment]
Effectivly, it works with version 5.0.3-rc.1.
May be patched in latest version. So the release of 5.0.3 will fix the issue.

**Comment by kamaloush** [comment]
Issue is patched since version 5.0.3
